### ISSUE TITLE
QuestionAboutGradients

### ISSUE THREAD

**Issue by tanakataiki2** [Unity-Technologies/ml-agents#1236]
Hello, I want to know how to set hyperparameter of gradient step size for updating value and policy.

so far loss is minimized by optimizer.minimize(self.loss) that is derived from value and policy loss and entrophy.
but how can I set updating step size for value and loss independently?

**Comment by awjuliani** [comment]
Hi @tanakataiki2,

The `learning_rate` parameter controls the overall gradient size. If you'd like control over individual strength of the components such as value and policy loss, I would recommend multiplying them by different values in the line creating the loss function.

**Comment by tanakataiki2** [comment]
@awjuliani 
Thank you very much for your reply
so if my goal is 
learning rate    = 1e^-4
value step size = 1e^-3
policy step size= 1e^-6
Then
loss = policy_loss*1e^-2 +  value_loss*10+ entrophy~~~?
or 
should I change to 
minimize(policy loss)
minimize(value loss)
minimize(entrophy)
